packages/@webex/internal-plugin-metrics/test/unit/spec/call-diagnostic/call-diagnostic-utils.ts

```diff
@@ -0,0 +1,75 @@
+import {assert} from '@webex/test-helper-chai';
+import {anonymizeIPAddress, clearEmpty, userAgentToString} from '../../../../src/call-diagnostic/call-diagnostic-metrics.util';
+import sinon from 'sinon';
+import * as anonymize from 'ip-anonymize';
+
+
+describe('internal-plugin-metrics', () => {
+  // describe('userAgentToString', () => {
```

Comment:
I can't seem to be able to mock modules...

Reply:
yeah looks like it's a pain, I've found this: https://minaluke.medium.com/how-to-stub-spy-a-default-exported-function-a2dc1b580a6b - so probably best solution is to just change BrowserDetection, but the ip-anonymize is an external lib. Anyway it's not a big deal, these are existing function that have worked fine for years, right? and they're not going to be changed often so we can live without the unit tests